src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt

```diff
@@ -38,11 +38,21 @@ class DebuggingPatch : ResourcePatch {
                         "revanced_debug_stacktrace_enabled",
                         StringResource(
                             "revanced_debug_stacktrace_enabled_title",
-                            "Print stack traces"
+                            "Debug log stack trace"
```

Comment:
Usually, we go with [Verb] [Noun] such as "Print stack traces", are you sure this is better?

Reply:
Maybe "debug logging stack trace"?

Reply:
If you want to indicate, that it applies ro logging, you can use "Log stack traces".
 out of the motivation that "Debuglogging" or "Debuglog" are not verbs.